BrowserKit/Sources/ComponentLibrary/Buttons/SecondaryRoundedButton.swift

```diff
@@ -5,16 +5,24 @@
 import Common
 import UIKit
 
-public class SecondaryRoundedButton: LegacyResizableButton, ThemeApplicable {
+public class SecondaryRoundedButton: ResizableButton, ThemeApplicable {
     private struct UX {
         static let buttonCornerRadius: CGFloat = 12
         static let buttonVerticalInset: CGFloat = 12
         static let buttonHorizontalInset: CGFloat = 16
         static let buttonFontSize: CGFloat = 16
+
+        static let contentInsets = NSDirectionalEdgeInsets(
+            top: buttonVerticalInset,
+            leading: buttonHorizontalInset,
+            bottom: buttonVerticalInset,
+            trailing: buttonHorizontalInset
+        )
```

Comment:
nice addition!

nit: Since we're not using the buttonInsets outside of `UX`, we can add the constants directly here and remove the `buttonHorizontalInset` and `buttonVerticalInset`

Reply:
What I like about this approach is to have a single constant for both top/bottom or left/right margins, since it's unlikely that we will ever use two different values for those. But if you prefer having the values set directly into this constant I see no objection 🙂 

If we decide to go this way it would be nice to also make that change in @thatswinnie's PR in order to keep things consistant between both buttons